pkg/plugins/plugins.go

```diff
@@ -237,3 +237,27 @@ func GetPluginMarkdown(pluginId string, name string) ([]byte, error) {
 	}
 	return data, nil
 }
+
+var coreDatasourceTypes = []string{
```

Comment:
Need to distinguish between core and other datasource. Not sure if there is better way to do this.